activesupport/lib/active_support/encrypted_configuration.rb

```diff
@@ -73,14 +83,23 @@ def validate! # :nodoc:
     #   # => { some_secret: 123, some_namespace: { another_secret: 789 } }
     #
     def config
-      @config ||= deserialize(read).deep_symbolize_keys
+      @config ||= deep_symbolize_keys(deserialize(read))
     end
 
     def inspect # :nodoc:
       "#<#{self.class.name}:#{'%#016x' % (object_id << 1)}>"
     end
 
     private
+      def deep_symbolize_keys(hash)
+        hash.deep_symbolize_keys
```

Comment:
@zzak @eileencodes it looks like this call to Hash#deep_symbolize_keys is not doing anything useful here and should be removed. Right?

Reply:
Whoops, yeah. Good catch. Do you want to open a PR? :pray: